volttrontesting/testutils/test_fixtures.py

```diff
@@ -9,4 +15,15 @@ def test_web_setup_properly(volttron_instance_web):
     assert instance.bind_web_address == instance.volttron_central_address
 
 
+@pytest.mark.skipif(SKIP_DOCKER, reason="No docker available in api (install pip install docker) for availability")
+def test_docker_wrapper():
+    with create_container("mysql") as container:
+        print(container.status)
+        print(container.logs())
```

Comment:
Unlike the test below, this only prints some attributes of `container`. Should it assert anything? On another note, would using the parametrize feature to test different images be useful here?